remote_write/README.md

```diff
@@ -32,7 +32,9 @@ The repo tests the following remote write senders:
 - The [Grafana Agent](https://github.com/grafana/agent).
 - [InfluxData's Telegraf](https://github.com/influxdata/telegraf).
 - The [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector).
-- The [VictoriaMetrics Agent](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/app/vmagent), unless you're on [Mac OS X](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1042).
+- The [VictoriaMetrics Agent](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/app/vmagent), unless you're on [Mac OS X]
```

Comment:
This removed both the actual URL part of the "Mac OS X" link, as well as the period at the end (plus added an extra empty line which isn't between the other entries).